spring-boot-starter/example-dmn-rest/pom.xml

```
@@ -6,7 +6,7 @@
   <version>0.0.1-SNAPSHOT</version>
 
   <properties>
-    <spring.boot.version>2.6.6</spring.boot.version>
+    <spring.boot.version>2.7.3</spring.boot.version>
```

```suggestion
    <spring.boot.version>2.7.4</spring.boot.version>
```
Can we use the latest spring boot 2.7 patch level?

---

As discussed, we will not bump the version any further here. For future releases let's always use the latest supported version.